styles/stylesheet.css

```diff
@@ -0,0 +1,458 @@
+    @media print {
+        body img {
+          display: none;
+       }
+    }
+    @media screen and (min-width: 200px) {
+        .homeimage {
+            height: 400px;
+            object-fit: contain;
+        }
+    }
+    @media screen and (min-width: 200px) {
+        .blogimage2 {
+          height: 200px;
+          object-fit: contain;
+          padding-top: 30px;
+        }
+    }
```

Comment:
I've noticed you have lots of duplicate @media queries. There is no limit on how much CSS you put into one query.

Reply:
Thanks! I tried to clean up my media queries some with assignment 7.